docs/source/index.rst

```diff
@@ -1,10 +1,14 @@
-.. Podman documentation master file, created by
-   sphinx-quickstart on Tue Oct 22 15:20:30 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. include:: includes.rst
 
-Welcome to Podman's documentation!
+What is Podman?
 ==================================
+Podman_ is a daemonless, open source, Linux native tool designed to make it easy to find, run, build, share and deploy applications using Open Containers Initiative (OCI_) Containers_ and `Container Images`_. Podman provides a command line interface (CLI) familiar to anyone who has used the Docker `Container Engine`_. Most users can simply alias Docker to Podman (`alias docker=podman`) without any problems. Similiar other common `Container Engines`_ (Docker, CRI-O, containerd), Podman relies on an OCI compliant `Container Runtime`_ (runc, crun, runv, etc) to interface with the operating system and create the running containers.This makes the running containers created by Podman nearly indistinguishable from those created by any other common container engine.
+
+Containers under the control of Podman can either be run by root or by a non-privileged user. Podman manages the entire container ecosystem which includes pods, containers, container images, and container volumes using the libpod_ library. Podman specializes in all of the commands and functions that help you to maintain and modify OCI container images, such as pulling and tagging. It allows you to create, run, and maintain those containers and container images in a production environment.
+
+The Podman service runs only on Linux platforms, however a REST API and clients are currently under development which will allow Mac and Windows platforms to call the service. There is currently a Varlink based remote client which runs on Mac or Windows platforms that allows the remote client to talk to the Podman server on a Linux platform. In addition to those clients, there is also a Mac client. NOTE: the Varlink remote client will be deprecated after the REST API is completed.
```

Comment:
after "under development" would it be worth adding a parenthical "(currently planned to be relase with Podman v2.0)" or something to that effect?

Reply:
As far as your note goes, @mheon and I went through a couple of review cycles for the [readme.md](https://github.com/containers/python-podman) on python-podman to come up with this note there:

```
Notice: The varlink interface to Podman is currently deprecated and in maintenance mode. When Podman version 2.0 is released in 2020, it will include a fully supported REST API that will replace the varlink interface. The varlink interface will be removed from Podman at a later release. Starting with Podman v1.8.0, the REST API is available for test usage. The documentation for the REST API resides here.
```